readers/api/transport.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 var (
-	errUnauthorizedAccess  = errors.New("missing or invalid credentials provided")
+	errEmptyToken          = errors.New("missing token")
```

Comment:
You can use `errors.ErrAuthentication` and remove this one

Reply:
@mteodor Please address this remark.